samples/dymaptic.GeoBlazor.Core.Sample.WebApp/dymaptic.GeoBlazor.Core.Sample.WebApp.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <UserSecretsId>8fd6e776-4434-4649-b202-09b0c3b479f5</UserSecretsId>
```

Comment:
dont know if you wanted this here

Reply:
Yep, that's fine. It's a server-based AspNetCore executable. When I've asked you to remove them is when they are not in the Executable project, or when they are in Wasm which doesn't support user secrets.